### ISSUE TITLE
Weird gold progress chart

### ISSUE THREAD

**Issue by ParanoikCZE** [Razviar/mtga-pro-tracker#142]
I was like to check my gold progress and seeing some gap even when I didn't buy anything before .. see 
![obrazek](https://user-images.githubusercontent.com/19796452/48286659-a4f33700-e466-11e8-88fe-a21de887c870.png)

Funny thing .. 18:00 value is before 17:00

![obrazek](https://user-images.githubusercontent.com/19796452/48286740-dec43d80-e466-11e8-9d97-0a40bb7428f0.png)

I'm sitting at central europe, so CET / GMT+1, not sure if that helps anyhow.

**Comment by Razviar** [comment]
We are switching from server time to user time, so such bugs are quite possible. Soon all tracked events will be bound to you timezone

**Comment by Spongman** [comment]
what happens when the machine travels between timezones?

**Comment by Razviar** [comment]
You data will be stored in that timezone, which is set up as current on
your machine

чт, 22 нояб. 2018 г., 20:51 Spongman notifications@github.com:

**Comment by Razviar** [comment]
You data will be stored in that timezone, which is set up as current on your machine

**Comment by Spongman** [comment]
right, but won't that cause an issue (like the one above) when the machine's timezone changes?
